taca/storage/storage.py

```diff
@@ -229,11 +229,11 @@ def _send_to_swestore(f, dest, remove=True):
             misc.call_external_command('iput -K -P {file} {dest}'.format(file=f, dest=dest),
                     with_log_files=True)
             logger.info('Run {} sent correctly and checksum was okay.'.format(f))
+            if remove:
+                logger.info('Removing run'.format(f))
+                os.remove(f)
```

Comment:
Are you deleting a directory? I think maybe `os.remove` will throw an exception if it's not empty; `shutil.rmtree`? Ignore me if I'm misunderstanding this.